substratevm/mx.substratevm/suite.py

```diff
@@ -961,7 +961,7 @@
                 "SVM",
                 "compiler:GRAAL_LLVM"
             ],
-            "maven" : False,
+            "maven" : True,
```

Comment:
This will not change what is published on Maven central: it deploys only to our internal repo. Do you need it there?

Reply:
Our plugins currently rely on this being available in maven local, but we can change that.